pkg/annotations/annotations.go

```diff
@@ -314,13 +314,13 @@ const (
 )
 
 // AnnotationExpansions maps annotations that will be expanded into an array of
-// other annotations. The expanded annotations will have the same value as the
-// original. It is an error for the expansions to already exist and have a value
-// that differs from the original.
-var AnnotationExpansions = map[string][]string{
+// other annotations and their values. It is an error for the expansions to already
+// exist and have a value that differs from the value set here.
+var AnnotationExpansions = map[string]map[string]string{
```

Comment:
since we are already changing the type, which is breaking, can we make it:

```suggestion
func AnnotationExpansions() map[string]map[string]string {
```

that way, no-one can (inadvertently) modify the map

Reply:
Could we just make it a const instead?